images/techdocs/context/Dockerfile

```diff
@@ -30,11 +30,14 @@ RUN ln -s /usr/local/lib/node_modules/corepack/dist/corepack.js /usr/local/bin/c
     && ln -s /usr/local/lib/node_modules/npm/bin/npm-cli.js /usr/local/bin/npm \
     && ln -s /usr/local/lib/node_modules/npm/bin/npx-cli.js /usr/local/bin/npx
 
+ENV JDK_VERSION=17
+ENV JDK_PACKAGE_VERSION=17.0.13+11-2~deb12u1
```

Comment:
This is the package we are replacing, we might have to move to bookworm at the same time. I'm getting this error when trying that package

```
 > [base 11/11] RUN apt-get update && apt-get install --no-install-recommends -y     openjdk-17-jdk-headless=17.0.12+7-2~deb11u1     openjdk-17-jdk=17.0.12+7-2~deb11u1     openjdk-17-jre-headless=17.0.12+7-2~deb11u1     openjdk-17-jre=17.0.12+7-2~deb11u1     && apt-get clean     && rm -rf /var/lib/apt/lists/*:
2.312 is only available from another source
2.312
2.312 Package openjdk-17-jre is not available, but is referred to by another package.
2.312 This may mean that the package is missing, has been obsoleted, or
2.312 is only available from another source
2.312
2.313 E: Version '17.0.12+7-2~deb11u1' for 'openjdk-17-jdk-headless' was not found
2.313 E: Version '17.0.12+7-2~deb11u1' for 'openjdk-17-jdk' was not found
2.313 E: Version '17.0.12+7-2~deb11u1' for 'openjdk-17-jre-headless' was not found
2.313 E: Version '17.0.12+7-2~deb11u1' for 'openjdk-17-jre' was not found
`` `

Reply:
Bump to bookworm to resolve this, image builds, testing now